src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -281,23 +281,27 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
         case gjre: GoogleJsonResponseException if gjre.getDetails.getCode == StatusCodes.Conflict.intValue => None
       }
       _ <- creationOperationId match {
-        case Some(opId) => pollShellProjectCreation(opId) //poll until it's created
+        case Some(opId) => for {
+          _ <- pollShellProjectCreation(opId) //poll until it's created
+          enableStorageOpId <- googleServiceManagerDAO.enableService(projectName, "storage-api.googleapis.com")
+          _ <- pollServiceOperation(enableStorageOpId) //poll until it's enabled
+        } yield ()
         case None => Future.successful(())
       }
       key <- getPetServiceAccountKey(user, GoogleProject(projectName))
     } yield key
   }
 
-  private def pollShellProjectCreation(operationId: String): Future[Boolean] = {
-    def whenCreating(throwable: Throwable): Boolean = {
-      throwable match {
-        case t: WorkbenchException => throw t
-        case t: Exception => true
-        case _ => false
-      }
+  private def whenOperationActive(throwable: Throwable): Boolean = {
+    throwable match {
+      case t: WorkbenchException => throw t
+      case t: Exception => true
```

Comment:
I'm worried about exceptions being swallowed by this statement.  Can you be more specific on the types of exceptions this is catching.  Maybe it is what you want, but maybe a quick log statement or something that we have an exception that we're not doing anything with.  At the very least, I would at least comment this line justifying why it's OK.